atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/Containers.java

```diff
@@ -88,6 +88,7 @@ public void before() throws Throwable {
 
             setupLogCollectorForLogDirectory();
             setupDockerComposeRule();
+            startCollectingLogs();
```

Comment:
I don't really like the multi-staged setup we've always had here, but I guess the kinds of things you're technically allowed to do by the API here are pretty broad, and this isn't really the scope of this PR anyway.

Reply:
We can discuss this in person if we want to change this in a subsequent PR.